nkms/crypto/encrypting_keypair.py

```diff
@@ -71,6 +71,19 @@ def encrypt(self,
         return ((ec.serialize(ekey.ekey), None),
                 cipher.encrypt(data))
 
+    def _decrypt_refactored_for_posterity(self,
+                                          combined_critical_mass, encrypted_message,
```

Comment:
Ah, I see™. You pretty much replaced ugly indexing [0] and [1] with human-readable names. Great